src/Main.java

```diff
@@ -8,45 +8,46 @@ public static void leapYear(int year){
         }
     }
 
-    public static void phoneVersion (short clientOS, short clientDeviceYear){
+    public static void versionPhone (short clientOS, short clientDeviceYear){
 
-        if (clientOS == 1 && clientDeviceYear > 2015) {
+        if (clientOS == 1 && clientDeviceYear > 2023) {
             System.out.println("Установите версию приложения для Android по ссылке.");
         }
 
-        if (clientOS == 0 && clientDeviceYear > 2015) {
+        if (clientOS == 0 && clientDeviceYear > 2023) {
             System.out.println("Установите версию приложения для iOS по ссылке.");
         }
-        if (clientOS == 1 && clientDeviceYear < 2015){
+        if (clientOS == 1 && clientDeviceYear < 2023){
             System.out.println("Установите облегченную версию приложения для Android по ссылке ");
         }
-        if (clientOS == 0 && clientDeviceYear < 2015) {
+        if (clientOS == 0 && clientDeviceYear < 2023) {
             System.out.println("Установите облегченную версию приложения для iOS по ссылке");
         }
     }
 
-    public static void bankCard(short deliveryDistance) {
-
+    public static int cardBank(short deliveryDistance) {
```

Comment:
```suggestion
    public static int calculateDeliveryDays(short deliveryDistance) {
```
сразу понятно по названию, что метод делает